scaleway/resource_storage_object_bucket.go

```diff
@@ -0,0 +1,108 @@
+package scaleway
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceScalewayStorageObjectBucket() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalewayStorageObjectBucketCreate,
+		Read:   resourceScalewayStorageObjectBucketRead,
+		Update: resourceScalewayStorageObjectBucketUpdate,
+		Delete: resourceScalewayStorageObjectBucketDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The name of the bucket",
+			},
+			"acl": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "private",
+				Description: "ACL of the bucket: either 'public-read' or 'private'. Private by default.",
```

Comment:
Is it a good practice to document the default value in the description? I can't find this practice in other providers...

> Description is used as the description for docs or asking for user input. It should be relatively short (a few sentences max) and should be formatted to fit a CLI.

Reply:
Probably not :p

Reply:
IIRC the default value is shown in the log and in the terraform schema while editing the file. 🤔